data/ecosystems/i/iotex.toml

```diff
@@ -1085,3 +1124,126 @@ url = "https://github.com/machinefi/w3bstream_compilets_demo"
 
 [[repo]]
 url = "https://github.com/machinefi/waas-flow-sdk-doc"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-client-python"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-client-go"
+
+[[repo]]
+url = "https://github.com/machinefi/western-chain-ws"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-client-js"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-client-arduino"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-docs-gitbook"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-client-esp32"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-http-client-simulator"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-v2"
+
+[[repo]]
+url = "https://github.com/machinefi/psa-crypto-arduino"
+
+[[repo]]
+url = "https://github.com/machinefi/psa-crypto-esp32"
+
+[[repo]]
+url = "https://github.com/machinefi/create-ws-app"
+
+[[repo]]
+url = "https://github.com/machinefi/web3-iot-sdk"
+
+[[repo]]
+url = "https://github.com/machinefi/google-fit-demo"
+
+[[repo]]
+url = "https://github.com/machinefi/oura-demo-v2"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-vscode-extension-assemblyscript"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-vscode-extension-go"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-vscode-extension-rust"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-studio-1"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-perf"
+
+[[repo]]
+url = "https://github.com/machinefi/test-required-workflow"
+
+[[repo]]
+url = "https://github.com/machinefi/.github"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/mimo-dao-contracts"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/ghost"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/mimo-bootstrap-contracts"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/mimo-intial-liquidity-contracts"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/mimo-docs"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/DefiLlama-Adapters"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/media-kits"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/mimo-toolkit-v2"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/mimo-swap-sdk"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/mimo2-contracts"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/mimo-sdk"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/pancake-farm"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/snapshot"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/pancake-toolkit"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/mysql"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/mimo-analytics"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/mimox-contracts"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/mimo-automator"
+
+[[repo]]
+url = "https://github.com/mimoprotocol/mimo-contract"
```

Comment:
you can remove all the mimoprotocol repos because `Mimo Defi` is a sub ecosystem already part of IoTeX and therefor we are already accounting for the activity of all its repos and attributing it to IoTeX